forms/FormAction.php

```diff
@@ -100,7 +100,7 @@ public function Title() {
 
 		// Remove this method override in 4.0
 		$decoded = Convert::xml2raw($title);
-		if($decoded !== $title) {
+		if($decoded != $title) {
```

Comment:
Why the loose equality? Could we cast title to string if it might not be a string instead?